tests/EmailAddress.test.ts

```diff
@@ -10,9 +10,6 @@ const valid: ReadonlyArray<string> = [
   'x@x.au',
   'foo@bar.com.au',
   'foo+bar@bar.com',
-  'hans.m端ller@test.com',
```

Comment:
Are these actually invalid?  I think these came from validators js originally

Reply:
unquoted, yeah; from what I can tell reading the rfc, unquoted is only "printable US-ASCII characters not including specials"

https://datatracker.ietf.org/doc/html/rfc5322#section-3.2.3